docs/content/guides/getting-started/demo.md

```diff
@@ -12,21 +12,36 @@ tags:
 
 [[toc]]
 
+::: only-for javascript
 Play around with a demo of Handsontable, in your favorite framework.
+:::
+
+::: only-for react
+Play around with a demo of Handsontable, in React.
+:::
+
+<div class="example-container">
+  <div id="example"></div>
+  <script src="https://examples.handsontable.com/examples/12.1.1/docs/js/demo/src.3f3ef8b8.js"></script>
```

Comment:
The urls for the `js` and `css` files are hardcoded for `12.1.1`, it'd be a good idea to use the version variable here, but I'm not sure if it's currently supported?

Reply:
@jansiegel Any idea how to do that?

Even if we use the version number from a variable, the filename part is also hard to determine. If I'm not wrong, it is content-hashed, so it will change when we change the demo.